lib/src/model/challenge/challenge.dart

```diff
@@ -40,10 +40,6 @@ abstract mixin class BaseChallenge {
 class Challenge with _$Challenge, BaseChallenge implements BaseChallenge {
   const Challenge._();
 
-  @Assert(
```

Comment:
as far as i can tell this doesnt break anything, but im not sure if some checks need to be done instead of this
thoughts @veloce

Reply:
needed the change becuase otherwise the challenge service breaks when trying to parse the list if there is an unlimited challenge in it.

Reply:
So an unlimited challenge will have no clock and no days, that's fine. I suppose then you could replace the check to ensure that both days and clock cannot be set at the same time.